docs/api/rows-examples.md

```diff
@@ -150,9 +154,9 @@ Response
 ```
 
 
-### 4. Update a Row
+### 4. Update Row(s) by ID
```

Comment:
Hello @thevahidal 

I think you need to provide an example of a bulk update.  I _presume_ the client provides an array of updated documents - but the API should demonstrate this, IMHO.